client/python/CONTRIBUTING.md

```diff
@@ -1,3 +1,4 @@
+=======
```

Comment:
Looks like you have merge conflicts.

Please remove any *.md files from this PR.  They are not part of this change.  

Please remove package-lock.json from toplevel repo.